From 001fe233196acbd4ce5d3efa6aacf81d4f3fa27c Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Thu, 5 Nov 2020 18:06:14 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368: remove regulator-xxx-microvolt
 for SWITCH pmic node

Switch is just a function of switching. There is no voltage setting
function. Voltage getting is the supply voltage.

Fixes: 3fc99e38fdbf ("regulator: resolve supply after creating
regulator")

Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Change-Id: Iee08b78b0f4361b7f8b18bea5b851050edb636fa
---
 arch/arm64/boot/dts/rockchip/rk3368-808.dtsi            | 2 --
 arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts | 2 --
 2 files changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi
index 8cdbd0d8f496..648ee673714d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-808.dtsi
@@ -598,8 +598,6 @@
 			vcc_sd: SWITCH_REG1 {
 				regulator-always-on;
 				regulator-boot-on;
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
 				regulator-name = "vcc_sd";
 				regulator-state-mem {
 					regulator-on-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts b/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts
index d38337741044..78bf30ba960e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-px5-evb-android.dts
@@ -677,8 +677,6 @@
 			vcc33_lcd: SWITCH_REG2 {
 				regulator-boot-on;
 				regulator-name = "vcc33_lcd";
-				regulator-min-microvolt = <3300000>;
-				regulator-max-microvolt = <3300000>;
 				regulator-state-mem {
 					regulator-off-in-suspend;
 				};
-- 
2.35.3

